language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/builder/impl/MakeNonAbstractObjectCodeAction.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.codeaction.builder.impl;
+
+import org.ballerinalang.langserver.codeaction.builder.DiagBasedCodeAction;
+import org.ballerinalang.langserver.common.constants.CommandConstants;
+import org.ballerinalang.langserver.commons.LSContext;
+import org.ballerinalang.langserver.commons.codeaction.LSCodeActionProviderException;
+import org.ballerinalang.langserver.compiler.DocumentServiceKeys;
+import org.ballerinalang.model.Whitespace;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4j.TextEdit;
+import org.wso2.ballerinalang.compiler.tree.BLangPackage;
+import org.wso2.ballerinalang.compiler.tree.BLangTypeDefinition;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Matcher;
+
+import static org.ballerinalang.langserver.codeaction.providers.AbstractCodeActionProvider.createQuickFixCodeAction;
+
+/**
+ * Code Action for making object non abstract type.
+ *
+ * @since 1.2.0
+ */
+public class MakeNonAbstractObjectCodeAction implements DiagBasedCodeAction {
```

Comment:
Need to disable this code action since we do not have abstract objects anymore. Let's keep this disabled until we finalise the object and class related code actions

Reply:
Yes, these are already disabled by the code-action provider.